packages/server/database/migrations/20221107180234-adjustUserTiers.ts

```diff
@@ -0,0 +1,80 @@
+import {R, RDatum, RValue} from 'rethinkdb-ts'
+import getPg, {closePg} from '../../postgres/getPg'
+import {TierEnum} from '../types/Invoice'
+import OrganizationUser from '../types/OrganizationUser'
+import User from '../types/User'
+
+export const up = async function (r: R) {
+  const removedOrgUserIds = await r
+    .table('OrganizationUser')
+    .hasFields('removedAt')
+    .filter((row) => row('tier').ne('personal'))('userId')
+    .distinct()
+    .run()
+
+  await r
+    .table('User')
+    .getAll(r.args(removedOrgUserIds))
+    .update(
```

Comment:
+1 Did you check if you could just return the changes to omit the next query in l43?